update.sh

```diff
@@ -154,8 +154,8 @@ for version in "${versions[@]}"; do
 		esac
 
 		# https://bugs.python.org/issue32598 (Python 3.7.0b1+)
-		# TL;DR: Python 3.7+ uses OpenSSL functionality which LibreSSL doesn't implement (yet?)
-		if [[ "$version" == 3.7* ]] && [[ "$variant" == alpine* ]]; then
+		# TL;DR: Python 3.7+ uses OpenSSL functionality which LibreSSL 2.6.x in Alpine 3.7 doesn't implement
+		if [[ "$version" == 3.7* ]] && [[ "$variant" == alpine3.7 ]]; then
```

Comment:
Should be `alpine3.8` instead of `alpine3.7`, isn't it?

Reply:
@lorddaedra  It isn't. On `alpine3.8` we can use `libressl` , the package's version is 2.7.x.

Reply:
looks like, you are right...

then we should merge this PR...

do we have any reasons to wait for anything?.. last commit was added 6 days ago...

Reply:
looks like, you are right...

then we should merge this PR...

do we have any reasons to wait for anything?..